proto/cosmos/tx/v1beta1/tx.proto

```diff
@@ -217,7 +217,7 @@ message Fee {
 // Tip is the tip used for meta-transactions.
 message Tip {
   // amount is the amount of the tip
-  repeated cosmos.base.v1beta1.Coin amount = 1 [(gogoproto.nullable) = false];
+  repeated cosmos.base.v1beta1.Coin amount = 1 [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
```

Comment:
should we avoid using cast repeated as it seems we may stop using it later on?

Reply:
sure... I guess we can just wrap it in `Coins` when needed